lib/rules/v-if-else-key.js

```diff
@@ -222,44 +253,55 @@ module.exports = {
         const currentScope = componentUsageStack[componentUsageStack.length - 1]
         const usageInfo = currentScope.get(componentName) || {
           count: 0,
-          firstNode: null
+          firstNode: null,
+          hasIf: false
         }
 
-        if (hasConditionalDirective(node)) {
-          // Store the first node if this is the first occurrence
-          if (usageInfo.count === 0) {
-            usageInfo.firstNode = node
-          }
+        const isIfDirective = utils.getDirective(node, 'if') !== null
+        const isConditional =
+          isIfDirective ||
+          utils.getDirective(node, 'else-if') !== null ||
+          utils.getDirective(node, 'else') !== null
 
-          if (usageInfo.count > 0) {
-            const uniqueKey = `${casing.kebabCase(componentName)}-${
-              usageInfo.count + 1
-            }`
-            checkForKey(
-              node,
-              context,
-              componentName,
-              uniqueKey,
-              conditionalFamilies
-            )
-
-            // If this is the second occurrence, also apply a fix to the first occurrence
-            if (usageInfo.count === 1) {
-              const uniqueKeyForFirstInstance = `${casing.kebabCase(
-                componentName
-              )}-1`
+        if (isConditional) {
+          if (isIfDirective && usageInfo.hasIf) {
+            // Reset if family already has an 'if' directive
+            usageInfo.count = 1
+            usageInfo.firstNode = node
```

Comment:
Will this work for the following case? Please also add this as a test case:

```vue
<ComponentA v-if="foo" />
<ComponentA v-else />

<ComponentA v-if="bar" />
<ComponentA v-else-if="baz" />
<ComponentA v-else />
```

An error should be reported for all of them.

Reply:
Currently, an error is reported for all of them except the second `if`